### PR TITLE
added env variable for user delete flow for ml services

### PR DESCRIPTION
**Pull request by ankit1shahu**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/ml-survey-service.env [hunk 1]
```diff
@@ -65,3 +65,18 @@ KEYCLOAK_PUBLIC_KEY_PATH={{ml_survey_keycloak_public_key_path|default("keycloak-
...
+USER_DELETE_ON_OFF={{user_delete_on_off | default("ON")}}
+
+#Topic name for user delete event consumer Topic Name
+USER_DELETE_TOPIC={{user_delete_topic_name | default(env_name+".delete.user")}}
+
+#ml core service id
+ID={{ml_core_service_id | default(env_name+"."+sunbird_instance+".ml.survey.service")}}
+
+#Telemetry enable/disbale service 
+TELEMETRY_ON_OFF={{telemetry_on_off | default("ON")}}
```
Comment (aks30) [inline ansible/roles/stack-sunbird/templates/ml-survey-service.env]: @ankitpws Rename - telemetry_on_off to manage_learn_telemetry_on_off

File: ansible/roles/stack-sunbird/templates/ml-survey-service.env [hunk 2]
```diff
@@ -65,3 +65,18 @@ KEYCLOAK_PUBLIC_KEY_PATH={{ml_survey_keycloak_public_key_path|default("keycloak-
 
 ## portal url of env
 APP_PORTAL_BASE_URL={{ proto }}://{{ domain_name }}
+
+#User Delete on and off flow
+USER_DELETE_ON_OFF={{user_delete_on_off | default("ON")}}
```
Comment (aks30) [inline ansible/roles/stack-sunbird/templates/ml-survey-service.env]: @ankitpws - Rename it to manage_learn_ user_delete_on_off

Comment (aks30) [reply ansible/roles/stack-sunbird/templates/ml-survey-service.env]: In this and above file

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 28 Nov

**Review by aks30 — APPROVED** [review]
Approved.

**Review by gandham-santhosh — APPROVED** [review]

**Pull request merged**